### ISSUE TITLE
AppInsightsLogger malformed XML

### ISSUE THREAD

**Issue by GeorgeTsiokos** [Microsoft/ApplicationInsights-dotnet#661]
_Ensure "Enable Just My Code" is not enabled_

## Repro Steps
1. Create an instance of the `AppInsightsLogger`:

```c#
new AppInsightsLogger("sdfsdf");
```

##  Actual Behavior
```
System.Xml.XmlException occurred
  HResult=0x80131940
  Message=Name cannot begin with the ' ' character, hexadecimal value 0x20. Line 1, position 136.
  Source=System.Xml
  StackTrace:
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
```

AppInsightsLogger constructor has a malformed XML constant:
```c#
XDocument.Parse("<?xml version=\"1.0\" encoding=\"utf-8\"?><ApplicationInsights xmlns = \"http://schemas.microsoft.com/ApplicationInsights/2013/Settings\"></ ApplicationInsights >");
```

## Expected Behavior
No First Chance Exception

## Version Info 
SDK Version :  `<package id="Microsoft.ApplicationInsights.SnapshotCollector" version="1.1.0" targetFramework="net47" />`
.NET Version :	4.7
How Application was onboarded with SDK(VisualStudio/StatusMonitor/Azure Extension) : `N/A`
OS : `N/A`
Hosting Info (IIS/Azure WebApps/ etc) : `N/A`

**Comment by SergeyKanzhelev** [comment]
What is `AppInsightsLogger`?

**Comment by GeorgeTsiokos** [comment]
A type used by the AI SnapshotCollector. Reference that package & create an instance, without just my code, break on 1st chance.

**Comment by SergeyKanzhelev** [comment]
@pharring can you please take a look? 

Also where can I file bugs on it? Why snapshot collector exposes `ILogger` implementation as a public class? And uses `AppInsights` instead of `ApplicationInsights`.

**Comment by GeorgeTsiokos** [comment]
Repro:
 - Add `Microsoft.ApplicationInsights.SnapshotCollector`
```c#
[TestMethod]
public void TrackTrace()
{
    using (new FirstChanceExceptionObserver())
    {
        var telemetryClient = new TelemetryClient(_telemetryConfiguration);
        telemetryClient.TrackTrace("Test");
        telemetryClient.Flush();
    }
}
```
With:
```c#
public sealed class FirstChanceExceptionObserver : IDisposable
{
    readonly ConcurrentBag<Exception> _exceptions = new ConcurrentBag<Exception>();

    public FirstChanceExceptionObserver() => AppDomain.CurrentDomain.FirstChanceException += OnFirstChanceException;

    public void Dispose()
    {
        AppDomain.CurrentDomain.FirstChanceException -= OnFirstChanceException;
        CheckForExceptions();
    }

    void CheckForExceptions()
    {
        switch (_exceptions.Count)
        {
            case 0:
                return;
            case 1:
                throw _exceptions.First();
            default:
                throw new AggregateException(_exceptions);
        }
    }

    void OnFirstChanceException(object sender, FirstChanceExceptionEventArgs firstChanceExceptionEventArgs) => _exceptions.Add(firstChanceExceptionEventArgs.Exception);
}
```

**Comment by GeorgeTsiokos** [comment]
This appears to be related to a larger issue #682 of first chance exceptions being introduced during initialization.

**Comment by pharring** [comment]
Sorry for the long delay. I missed the ping.
Please don't use AppInsightsLogger. It should be an internal class.
It snuck into the public surface area of version 1.0.11.
It was made properly internal in 1.1.0

Having said that, you're right that there's a bug in that configuration string. I'll look into a fix for the next release.

**Comment by pharring** [comment]
This has been fixed in version 1.1.1 of the SnapshotCollector nuget.
